MapleServer2/Servers/Game/FieldManager.cs

```diff
@@ -530,20 +530,18 @@ private Task StartMapLoop()
 
         private void MonsterMovement()
         {
-            Random Rand = new Random();
+            Random rand = new Random();
             foreach (IFieldObject<Mob> mob in State.Mobs.Values)
             {
-                short x = (short) Rand.Next(-70, 70); //random x position, units are block units
+                short x = (short) rand.Next(-150, 150); //random x position, units are block units
```

Comment:
There's a variable for 150 in the source, I forgot where though

Reply:
`Block.cs`